### ISSUE TITLE
Function naming convention 

### ISSUE THREAD

**Issue by shadybassily** [javascript-tutorial/en.javascript.info#3652]
in Functions section, exactly inside naming conventions, you recommend to prefix a function with check... if the function returns a true/false, yet in one of the examples, you named a function isPrime that returns true if a num is prime.

**Comment by prakhar-pal** [comment]
I don't believe it matters that much as long as function names are meaningful.

This is a convention but conventions can change by project to project.
